### PR TITLE
Add CHANGELOG and update README for latest release

### PR DESCRIPTION
**Pull request by w01fe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## 7/13/2015
+ - **DEPRECATED** `/topic/recent-url` endpoint; prefer new `/doc/search` endpoint.
+ - added `/doc/search` to search for docs. see [blog](blog.getprismatic.com/feeds-api-and-interest-graph-explorer/) for intro and docs for query syntax
+ - added `/aspect` to list all aspects by aspect 'key' (the format expected by `/doc/search`). 
+ - improved performance of /topic/search
```
Comment (davegolland) [inline CHANGELOG.md]: :sheep: backticks `/topic/search`

Comment (loganlinn) [inline CHANGELOG.md]: :sheep: add ` around /topic/search like other endpoints mentioned here

File: README.md [hunk 2]
```diff
@@ -42,16 +40,16 @@ The service provides an endpoint for returning the set of topics that are simila
...
-signals.
+###Feeds API
+The service provides recent, high-quality URLs from all over the web for a given 
+query (which can include both topics and aspects), including extracted metadata
+for each URL.  The [Interest Graph Explorer](http://prismatic.github.io/explorer/)
+includes an interactive demo for the Feeds API.
 
-##Is the service still in ALPHA?
+###What's new?
+We are working hard to continually extend and improve the functinoality of the Interest Graph API.
```
Comment (davegolland) [inline README.md]: typo: functionality

File: README.md [hunk 3]
```diff
@@ -42,16 +40,16 @@ The service provides an endpoint for returning the set of topics that are simila
...
 and reports the aspects, which describe the structure or function of the webpage.
 
-###Topic Feeds
-The service provides recent, high-quality URLs from all over the web for a given input
-topic. The recent URLs endpoint takes as input a topic from our Interest Graph,
-and returns a list of URLs about that topic that have recently been shared on
-social networks and pass a certain quality bar derived from social and content
-signals.
+###Feeds API
+The service provides recent, high-quality URLs from all over the web for a given
```
Comment (davegolland) [inline README.md]: :sheep: :sheep: high-quality documents as below?

File: README.md [hunk 4]
```diff
@@ -278,6 +292,8 @@ covered by the first two categories. Note: at the moment, content
 classification for NSFW aspects is determined solely based on the text and
 metadata of the web page -- not the imagery.
 
+You can also use the `/aspects` endpoint to programatically list the set of all currently
```
Comment (davegolland) [inline README.md]: typo: "programmatically"

**Comment by davegolland** [comment]
LGMC

File: README.md [hunk 5]
```diff
@@ -113,9 +127,9 @@ headers, you can alternatively pass it in a query parameter
 `?api-token=<API-TOKEN>`. Omitting the token from both the query parameter and
 header will result in a `401` status code from the server.
 
-While we are still in ALPHA, we rate limit requests by default.
-To request a rate limit increase, email <public-api@getprismatic.com>
-with details about your use case and desired rates per API endpoint.
+Requests are rate limited based on your service package.  Please see 
+[Developer page](http://getprismatic.com/developers) for the latest information,
```
Comment (loganlinn) [inline README.md]: "Please see *the* Developer page"

Comment (w01fe) [inline README.md]: oops, fixed.

**Pull request merged**